extensions/amp-video/0.1/test/test-amp-video.js

```diff
@@ -455,8 +455,10 @@ describes.realWin('amp-video', {
             return listenOncePromise(v, VideoEvents.PAUSE);
           })
           .then(() => {
-            impl.unmute();
-            return listenOncePromise(v, VideoEvents.UNMUTED);
+            // Can NOT test unmute because browsers no longer allow unmutting
+            // without user-interaction :(
+            // impl.unmute();
+            // return listenOncePromise(v, VideoEvents.UNMUTED);
```

Comment:
Will the `.then()` block after this fail as a result of not calling `impl.unmute()`?